client/starwhale/utils/config.py

```diff
@@ -108,10 +108,26 @@ def __init__(self, swcli_config: t.Optional[t.Dict[str, t.Any]] = None) -> None:
     def rootdir(self) -> Path:
         return Path(self._config["storage"]["root"])
 
+    @property
```

Comment:
remove `workdir`, `pkgdir` and `dataset_dir`? these properties are deprecated.  the children dir in the rootdir are the project dirs.

related pr: https://github.com/star-whale/starwhale/pull/902

Reply:
done